schemas/threat.yml

```diff
@@ -0,0 +1,70 @@
+---
+- name: threat
+  title: Threat
+  group: 2
+  short: Fields describing detected threats and threat intel data
+  description: >
+    ECS threat fields encompass detected suspicious activity (e.g. IDS alert, malware activity) typically
+    reported by IDS/IPS and NGFWs, as well as threat intel/global IOC data ingested from a feed source like MISP or Anomali.
+
+  type: group
+  fields:
+
+    - name: id
+      level: extended
+      type: keyword
+      short: Threat ID
+      description: >
+        Threat ID, typically provided by reporting system or threat intel provider.
+      example: 33227 
+
+    - name: name
+      level: extended
+      type: keyword
+      short: Name or classification of the threat detected or reported
+      description: >
+        Threat name as reported by system or intel provider
+
+        The field value must be normalized to lowercase for querying. See
+        the documentation section "Implementing ECS".
+      example: ntp reserved mode denial of service
+
+    - name: tactic
+      level: extended
+      type: keyword
+      short: Mitre ATT&CK Matrix tactic
+      description: >
+          The Mitre ATT&CK Matrix Tactic categorization of the identified threat 
```

Comment:
@MikePaquette So do we want to go all in on Mitre ATT&CK and say that this field (and also the `.technique` field) are specifically the Mitre framework's tactic and technique?

Or should we make it optional? E.g.:

> The type of tactic used by this threat. You can use the Mitre ATT&CK Matrix Tactic categorization, for example.

My preference would be to make it optional.

Reply:
@webmat @dainperkins I think ECS should not "require" the MITRE ATT&CK framework, but should allow it, and as @webmat suggests, making this explicit in the descriptions of the fields.